solution/backend/resources/views/mixins.py

```diff
@@ -72,7 +72,6 @@ def get_location_filter(self, locations):
 class LocationExplorerViewSetMixin(OptionalPaginationMixin, LocationFiltererMixin):
     PARAMETERS = LocationFiltererMixin.PARAMETERS + OptionalPaginationMixin.PARAMETERS
 
-    location_filter_prefix = ""
     location_filter_max_depth = 2
```

Comment:
wasnt letting file upload overload the location filter prefix.  Removing it fixes it.